packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -30,63 +31,81 @@ export function packageDefaults(version?: string, description?: string): Partial
  * @param {App} app - specifies the application properties
  * @returns {Partial<App>} the App instance
  */
-export function appDefaults(app: App): App {
-    return {
-        version: app.version || '0.0.1',
-        id: app.id,
-        uri: app.id.replace('.', '/'), // todo: remove if unused
-        title: app.title || `Title of ${app.id}`, //todo: localise
-        description: app.description || `Description of ${app.id}`, //todo: localise
-        baseComponent: app.baseComponent || 'sap/ui/core/UIComponent',
-        flpAppId: app.flpAppId || `${app.id.replace(/[-_.]/g, '')}-tile`
-    };
+export function mergeApp(app: App): App {
+    // Return merged, does not update passed ref
+    return Object.assign(
+        {
+            version: '0.0.1',
+            title: `Title of ${app.id}`, //todo: localise
+            description: `Description of ${app.id}`, //todo: localise
+            baseComponent: 'sap/ui/core/UIComponent'
+        },
+        app
+    ) as App;
 }
 
-export const UI5_DEFAULT = {
-    DEFAULT_UI5_VERSION: '',
-    DEFAULT_LOCAL_UI5_VERSION: '1.95.0',
-    MIN_LOCAL_SAPUI5_VERSION: '1.76.0',
-    MIN_LOCAL_OPENUI5_VERSION: '1.52.5',
-    SAPUI5_CDN: 'https://ui5.sap.com',
-    OPENUI5_CDN: 'https://openui5.hana.ondemand.com'
-};
+export enum UI5_DEFAULT {
+    DEFAULT_UI5_VERSION = '',
+    DEFAULT_LOCAL_UI5_VERSION = '1.95.0',
+    MIN_UI5_VERSION = '1.60',
```

Comment:
```suggestion
    MIN_UI5_VERSION = '1.60.0',
```
patch version here also?

Reply:
In that case it doesn't make a difference but I didn't see a reason why not being specific here.